hazelcast/src/test/java/com/hazelcast/config/alto/AltoConfigTestUtil.java

```diff
@@ -27,4 +28,13 @@ public static AltoSocketConfig getClientSocketConfig(HazelcastInstance hz) {
     public static List<Integer> getClientPorts(HazelcastInstance hz) {
         return getTpcServerBootstrap(hz).getClientPorts();
     }
+
+    public static void assertClientPorts(int eventloopCount, int startInclusive, int endInclusive,
+                                         HazelcastInstance... hz) {
+        for (HazelcastInstance i : hz) {
```

Comment:
I'm sure we can come up with better names than 'i'. This is typically reserved for an index of a loop.

Reply:
'i' :)